app/helpers/number_helper.rb

```diff
@@ -69,8 +69,6 @@ def mhz_to_human_size(size, *args)
     nil
   end
 
-  private
-
```

Comment:
So, it turns out, it can't be private anyway because we call `NumberHelper.handling_negatives`, in other words, with a receiver.  Since it was always public, keeping it public.